oligo_designer_toolsuite/oligo_specificity_filter/_filter_blastn.py

```diff
@@ -371,12 +634,30 @@ def __init__(
         blast_search_parameters: dict = {},
         blast_hit_parameters: dict = {},
         dir_output: str = "output",
+        names_search_output: list = [
+            "query",
+            "reference",
+            "alignment_length",
+            "query_start",
+            "query_end",
+            "query_length",
+            "query_sequence",
+            "reference_start",
+            "reference_end",
+            "reference_sequence",
+            "reference_strand"
+        ],
+        ai_filter: str = None,
+        ai_filter_threshold: float = None,
+        ai_filter_path: str = None,
```

Comment:
same as above. move ai filters to own class and don't use all names_search_output per default but only keep minimum required ones for pure blast filter